gen/generate_test.go

```diff
@@ -21,16 +21,32 @@
 package gen
 
 import (
+	"errors"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/thriftrw/thriftrw-go/compile"
+	"github.com/thriftrw/thriftrw-go/internal/plugin"
+	"github.com/thriftrw/thriftrw-go/internal/plugin/handletest"
+	"github.com/thriftrw/thriftrw-go/plugin/api"
+	"github.com/thriftrw/thriftrw-go/plugin/plugintest"
 
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func testdata(t *testing.T, paths ...string) string {
+	cwd, err := os.Getwd()
```

Comment:
by default all paths are relative to `cwd` so do you need to append this or can you just append to `./testdata`